### PR TITLE
Rename variable names for better readibility

### PR DESCRIPTION
**Pull request by cydrain**
Signed-off-by: yudong.cai <yudong.cai@zilliz.com>

### CODE REVIEW

**Comment by cydrain** [comment]
rerun ut

File: internal/core/src/query/generated/ShowExprVisitor.h [hunk 1]
```diff
@@ -38,16 +38,13 @@ class ShowExprVisitor : public ExprVisitor {
     visit(CompareExpr& expr) override;
 
  public:
-    using RetType = Json;
-
- public:
-    RetType
+    Json
```
Comment (xiaofan-luan) [inline internal/core/src/query/generated/ShowExprVisitor.h:41]: why some of the function return Json while others return RetType

**Pull request merged**